docs/configure/user-interface/sidebar-and-urls.mdx

```diff
@@ -113,3 +113,46 @@ When Storybook generates the titles for all matching stories, they'll retain the
 
 You can provide your own indexer to include stories with a different naming convention, adjust the automatic title generation beyond a prefix, and many other use cases. For more information, see the [Story Indexers API reference](../../api/main-config/main-config-indexers.mdx).
+
+## Sidebar UI
+
+### Conditionally show sidebar
+
+By default, the sidebar is visible on every page, and users can choose to hide it with a keyboard shortcut.
+
+If you need, you can customize when the sidebar appears. For example, this configuration will always hide it on a MDX page with title `'Landing'`:
+
+{/* prettier-ignore-start */}
+
+<CodeSnippets
+  paths={[
+    'storybook-manager-sidebar-hide-on-landing.md',
+  ]}
+/>
+
+{/* prettier-ignore-end */}
+
+<Callout variant="warning">
+  The `showSidebar` function lets you hide the primary navigation mechanism of the Storybook UI. When hiding the sidebar, make sure the page being displayed provides an alternative means of navigation.
+</Callout>
+
+
+The following parameters are available to the `showSidebar` function:
+
+
+| Name                     | Type     | Description                                                    | Example Value                         |
+| ------------------------ | -------- | -------------------------------------------------------------- | ------------------------------------- |
+| **path**                 | String   | Path to the page being displayed                               | `'/story/components-button--default'` |
+| **viewMode**             | String   | Whether the current page is a story or docs                    | `'docs'` or `'story'`                 |
+| **singleStory**          | Boolean  | Whether the current page is the only story for a component     | `true` or `false`                     |
+| **storyId**              | String   | The id of the current story or docs page                       | `'blocks-blocks-unstyled--docs'`      |
+| **layout**               | Object   | The current layout state                                       | *see below*                           |
+| **layout.isFullscreen**  | Boolean  | Whether the preview canvas is in fullscreen mode               | `true` or `false`                     |
+| **layout.panelPosition** | String   | Whether the panel is shown below or on the side of the preview | `'bottom'` or `'right'`               |
+| **layout.showNav**       | Boolean  | The setting for whether the end user wants to see the sidebar  | `true` or `false`                     |
+| **layout.showPanel**     | Boolean  | The setting for whether the end user wants to see the panel    | `true` or `false`                     |
+| **layout.showSidebar**   | Boolean  | The setting for whether the end user wants to see the sidebar  | `true` or `false`                     |
+
+### Keyboard shortcuts
+
+- Show or hide sidebar: <kbd>s</kbd>
```

Comment:
@Sidnioulz, if you're okay with it, can you revert this back to what it was and consolidate the documentation in the features and behavior docs? Also, if you're around at the next triage meeting, I would like to discuss a bit more with you the options in the tables and guidance on this pull request

Reply:
I've updated the branch with your change requests so we can talk it out tomorrow. Thanks for the review Jonnie!